Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:2.7-alpine
```

Comment:
Could you add a [healthcheck](https://docs.docker.com/engine/reference/builder/#healthcheck) to this image?